lua/entities/gmod_wire_expression2/core/cl_http_cl.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
```

Comment:
Having the author in a file is only meaningful until someone else changes stuff.
But let's talk about the merits of having or not having the author in there in the coding style dicussion :)